mne/beamformer/_lcmv.py

```diff
@@ -182,11 +170,23 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
         ch_names=ch_names, proj=proj, is_ssp=is_ssp, vertices=vertno,
         is_free_ori=is_free_ori, nsource=forward['nsource'], src_type=src_type,
         source_nn=forward['source_nn'].copy(), subject=subject_from,
-        rank=rank_int)
+        rank=rank_int, max_power_ori=max_power_ori)
 
     return filters
 
 
+def _proj_data(M, proj, filters):
+    if filters['is_ssp']:
+        # check whether data and filter projs match
+        _check_proj_match(proj, filters)
+        if filters['whitener'] is None:
```

Comment:
@wmvanvliet we use it here

`_proj_data` is perhaps not the best name, could rename to `_proj_whiten_data`